spec/lib/object_serializer_polymorphic_spec.rb

```diff
@@ -48,4 +75,13 @@ class ListSerializer
       expect(record_type).to eq 'car'.to_sym
     end
   end
+
+  context 'when serializing included polymorphic relationships' do
+    it "should not raise" do
+      list = List.new
+      list.id = 1
+      list.items = [checklist_item, car]
+      expect(-> {ListSerializer.new(list, include: ["items.list"])}).not_to raise_error
```

Comment:
RSpec syntax allows passing directly a block:
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher